keyboards/lily58/lib/layer_state_reader.c

```diff
@@ -4,14 +4,16 @@
 #include "lily58.h"
 
 #define L_BASE 0
-#define L_LOWER (1 << 1)
-#define L_RAISE (1 << 2)
-#define L_ADJUST (1 << 3)
+#define L_LOWER (1 << 2)
```

Comment:
These changes would break the current assumptions for the default keymaps and any user ones based off it.

Reply:
Rolled these back.